### PR TITLE
await setLang for FR16

### PR DESCRIPTION
**Pull request by blacksponge**
## Purpose

Some tests (FR16) fails, it looks like the lang has not been changed yet. Tries to await.

## Context

Following #223

### CODE REVIEW

**Review by matsduf — APPROVED** [review]
I think this looks fine.

**Comment by blacksponge** [comment]
After a more thorough check, is seems that the loading of this json was causing delays so I added a sleep. It would have been better to use `browser.wait` but I could figure what a reliable wait condition could be.

File: e2e/FR16.e2e-spec.ts [hunk 1]
```diff
@@ -16,12 +16,15 @@ describe('Zonemaster test FR16 - [The advanced view should have a text describin
     expect(element(by.css('.alert.alert-info')).element(by.css('a')).getAttribute('fragment')).toBe('q12');
   });
 
-  it('should have a description text in multi languages', () => {
-    utils.setLang('en');
+  it('should have a description text in multi languages', async () => {
+    await utils.setLang('en');
+    await browser.sleep(100);
```
Comment (matsduf) [inline e2e/FR16.e2e-spec.ts:21]: Is that seconds? Isn't 100 s beyond what is needed?

Comment (blacksponge) [reply e2e/FR16.e2e-spec.ts:21]: It is in milliseconds

Comment (matsduf) [reply e2e/FR16.e2e-spec.ts:21]: > It is in milliseconds

Then it is more than fine. :-)

**Review by matsduf — APPROVED** [review]

**Comment by mattias-p** [comment]
I'm going way out on a limb here but after #222, couldn't you do something like this? (Keep in mind that I have neither tried this myself, nor do I really know any of this stuff.)

```typescript
browser.wait(protractor.ExpectedConditions.presenceOf($('.lang > div > a.selected[lang="' + lang + '"]')), 5000);
```

Also, wouldn't it make sense to move the waiting into util.setLang()?

**Comment by blacksponge** [comment]
It should be possible to do something with `browser.wait`, I'll have another look into that.

**Comment by blacksponge** [comment]
The changes now live in #222, as they depend on the changes there.

**Comment by matsduf** [comment]
> The changes now live in #222, as they depend on the changes there. If that's okay we can close this PR.

Please update the description of #222 and refer to this PR.

**Comment by blacksponge** [comment]
> Please update the description of #222 and refer to this PR.

Done.

**Pull request closed without merging**